spec.yml

```diff
@@ -0,0 +1,34 @@
+requirements:
+  base:
+    image: debian:stretch
+    pkg-manager: apt
+  fsl:
+    version: 5.0.10
+  afni:
+    version: latest
+  miniconda:
+    conda_install: [python=3.7, nipype]
+
+nfm_test:
+  data:
+    type: workflow_path
+    location: test/data
+  scripts:
+    test/scripts
+  env:
+    base:
+    - {image: debian:stretch, pkg-manager: apt}
+    fsl:
+    - {version: 5.0.10}
+    afni:
+    - {version: latest}
+    miniconda:
+    - {conda_install: [python=2.7, nipype]}
+
+  analysis:
+    inputs:
+     - {type: File, value: sub-01_ses-test_T1w.nii.gz}
```

Comment:
Hmm. We might want to consider promoting datalad subdatasets for test files. Two unchanging multi-MB files isn't too bad, but if we ever do change these, the repository will quickly become unusably slow.

Reply:
definitely, this is why [data](https://github.com/djarecka/coco2019-unifize_and_skullstrip/blob/b13af899036e0bf31d9505555bc567eafa5b8bd2/spec.yml#L13) was added to the spec, so we can have datalad in `type`, but not implemented yet